test/e2e/apimachinery/resource_quota.go

```diff
@@ -1524,3 +1609,33 @@ func waitForResourceQuota(c clientset.Interface, ns, quotaName string, used v1.R
 		return true, nil
 	})
 }
+
+// updateResourceQuotaUntilUsageAppears updates the resource quota object until the usage is populated
+// for the specific resource name.
+func updateResourceQuotaUntilUsageAppears(c clientset.Interface, ns, quotaName string, resourceName v1.ResourceName) error {
+	return wait.Poll(framework.Poll, 1*time.Minute, func() (bool, error) {
+		resourceQuota, err := c.CoreV1().ResourceQuotas(ns).Get(quotaName, metav1.GetOptions{})
+		if err != nil {
+			return false, err
+		}
+		// used may not yet be calculated
+		if resourceQuota.Status.Used == nil {
```

Comment:
Well, here is the bug, when `status.used` is nil, we should keep updating `spec.hard`.